index.d.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Creates a Promise for either the project root path or null if it couldn't be found.
+ * 
+ * @param cwd Directory to start from. Default: process.cwd()
+ * @returns A Promise for either the project root path or null if it couldn't be found.
+ */
+export default function pkgDir(cwd?: string): Promise<string | null>;
```

Comment:
Is this actually a default export though? I just used this package and came up with the following for a declaration file:

```typescript
declare module 'pkg-dir' {
  function pkgDir(cwd?: string): Promise<string | null>;
  namespace pkgDir {
    function sync(cwd?: string): string | null;
  }
  export = pkgDir;
}
```

Reply:
No its not a default export, it has to be added to the index.js file.

Reply:
It's also in the styleguide.